src/main/java/me/xmrvizzy/skyblocker/skyblock/rift/TwinClawsIndicator.java

```diff
@@ -0,0 +1,42 @@
+package me.xmrvizzy.skyblocker.skyblock.rift;
+
+import me.xmrvizzy.skyblocker.config.SkyblockerConfig;
+import me.xmrvizzy.skyblocker.utils.SlayerUtils;
+import me.xmrvizzy.skyblocker.utils.Utils;
+import net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientTickEvents;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.sound.SoundEvent;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+import net.minecraft.util.Identifier;
+
+public class TwinClawsIndicator {
+    public static void init() {
+        ClientTickEvents.END_CLIENT_TICK.register(TwinClawsIndicator::updateIce);
```

Comment:
Same thing, is there a better event to hook into for this (perhaps world render events again) just so that the client isn't constantly ticking this